### ISSUE TITLE
Tensor board empty scalers

### ISSUE THREAD

**Issue by LoayHM** [tensorflow/ranking#20]
All the scalars that are in the tensorboard are almost empty, where they only show the either only the 1st or 100th step.

Here's a couple of examples of downloaded CSVs after running.

run_eval-tag-metric_ndcg@10

Wall time | Step | Value
--- | --- | ---
1547130262.059631 | 100 | 0.792011022567749
1547130264.828965 | 100 | 0.792011022567749
1547133669.9526756 | 100 | 0.792011022567749

run_.-tag-loss

Wall time | Step | Value
--- | --- | ---
1547130254.665982 | 1 | 1.5420621633529663

I ran the code as in the instructions, without any edit, still I get those weird scalers.

The tf.summary namespace is included normally in the code, still gets ambiguous results.
`  tf.summary.scalar("input_sparsity", tf.nn.zero_fraction(input_layer))
      tf.summary.scalar("input_max", tf.reduce_max(input_layer))
      tf.summary.scalar("input_min", tf.reduce_min(input_layer))
`

I am running remotely on an ec2 instance, using `ssh -L 16006:127.0.0.1:6006 ...` and opening `http://localhost:16006/` on chrome.

**Comment by LoayHM** [comment]
solved: as the scalers are saved each 100 steps, increase the number of training steps through  --num_train_steps flag

**Comment by eggie5** [comment]
You are talking about the example libsvm script right? What does step size mean in this context? Is it 1 batch of data? How do you specify 1 epoch?

**Comment by LoayHM** [comment]
I believe in this case it means epoch
